videos/videos_router.js

```diff
@@ -33,105 +36,70 @@ router.get("/:id", async (req, res) => {
   }
 });
 
-router.get("/user/:userId", async (req, res) => {
```

Comment:
It looks like the path to find videos by the user was erased by accident possibly? 
```
router.get("/user/:userId", async (req, res) => {
  const { userId } = req.params;
  try {
    const userVideos = await Videos.findByUser(userId);
    res.status(200).json(userVideos);
  } catch (err) {
    res.status(500).json({ message: "Try again later.", err });
  }
}); 
```
I believe we are currently using this to post the user's videos to their profile page. I'll send up a commit with it added back into the file.

Reply:
Good Catch. Thanks.

Reply:
Turns out the getbyUserID route was on master. Its in this version for sure after the merge resolution.